trestle/transforms/utils/osco_helper.py

```diff
@@ -61,36 +61,32 @@
 class RuleUse():
     """Represents one rule of OSCO data."""
 
-    def __init__(
-        self,
-        id_: str,
-        target: str,
-        target_type: str,
-        benchmark_href: str,
-        benchmark_id: str,
-        scanner_name: str,
-        scanner_version: str,
-        idref: str,
-        version: str,
-        time: str,
-        result: str,
-        severity: str,
-        weight: str
-    ) -> None:
+    def __init__(self, **args) -> None:
```

Comment:
I can see what you have done here to circumvent sonar complaining about too many arguments (I think it's limit is 7?).

However, this is pretty ugly as all the arguments are still required and there is now no longer an explicit spec. 
One idea would be to pass these as a dict (after all there will be no overhead as that is what the function call is doing anyway here).

TL;DR: It looks strange.

Reply:
I think is is 12 or 13, I missed by 1. Fixed.